src/dialog-editor/components/toolbox/toolboxComponent.ts

```diff
@@ -9,7 +10,7 @@ class DialogField {
               options: any = {}) {
     this.icon = icon;
     this.label = label;
-    this.placeholders = Object.assign({
+    this.placeholders = (<any>Object).assign({
```

Comment:
(the method *is* defined in `./node_modules/typescript/lib/lib.es6.d.ts`, just not sure how to make it use that without using `target: "es6"`)